lambda/finalizer.js

```diff
@@ -27,11 +29,19 @@ const getStrategy = (event) => {
     return event[0].strategy || defaultStrategy;
 };
 
+const getBalancedWeight = (event) => {
+    // extract weight used by balanced strategy or fallback to default (0.5)
+    const w = ("balancedWeight" in event[0]) ?  event[0].balancedWeight : defaultBalancedWeight;
```

Comment:
Preferred syntax:

```javascript
const weight = event[0].balancedWeight || defaultBalancedWeight;
```

Reply:
That was the first thing I tried but later realized that it doesn't work when balancedWeight = 0

Reply:
Fixed